README.rst

```diff
@@ -108,6 +103,32 @@ or::
 if your compiler does not support gcc-style options, if you are cross
 compiling, etc.
 
+Additional Language Interfaces
+----------------------------------------
+
+The Fortran interface is built alongside the library automatically.
+
+Python users can install using::
+
+    pip install libceed
+
+or in a clone of the repository via ``pip install .``.
+
+Julia users can install using::
+
+    ] add LibCEED
+
+in the Julia package manager or in a clone of the repository via::
+
+    JULIA_LIBCEED_LIB=/path/to/libceed.so julia
+    julia> # press ] to enter package manager
+    (env) pkg> build LibCEED
+
+Rust users can install using::
+
+    LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/path/to/libceed.so
+    cd rust
+    cargo build
```

Comment:
This doesn't tell a user how to use it, just how to build it. We'd need to tell users what to put in their `Cargo.toml`. (Publishing to crates.io is still a todo.)

Reply:
How about the updated language?